backend/corpora/common/utils/regex.py

```diff
@@ -4,23 +4,23 @@
 ID_REGEX = r"[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}"
 DATASET_ID_REGEX = f"(?P<dataset_id>{ID_REGEX})"
 COLLECTION_ID_REGEX = f"(?P<collection_id>{ID_REGEX})"
-CURATOR_TAG_PREFIX_REGEX = r"(?P<tag_prefix>.*)"
 CONTROL_CHARS = r"[\x00-\x1f\x7f-\xa0]"
-CURATOR_TAG_REGEX = r"(?P<tag>.*)"
+CURATOR_TAG_REGEX = r"(?P<tag>.+)"
 
 
 def validate_curator_tag(curator_tag: str) -> bool:
     """
-    Verify the correct curator tag format is obeyed.
+    Verify the correct curator tag format is obeyed (i.e., it is not a UUID)
 
     :param curator_tag: the tag name to validate.
-    :return: True if CURATOR_TAG_PREFIX_REGEX is matched.
+    :return: True if CURATOR_TAG_REGEX is matched.
     """
     regex = f"^({DATASET_ID_REGEX}|{CURATOR_TAG_REGEX})$"
     matched = re.match(regex, curator_tag)
-    if matched and (tag := matched.groupdict().get("tag")):
-        if not re.search(ID_REGEX, tag):
```

Comment:
removing this because we are only in the business of making sure that we know when to treat an identifier as a uuid, and that is when the identifier takes the precise form of a uuid. Otherwise, if it is not exactly the form of a uuid, we treat it as a curator tag.

Reply:
This requirement was explicitly in the ticket when I implemented it. I updated the ticket to reflect the actual requirements.